lib/dest/index.js

```diff
@@ -12,25 +15,30 @@ function dest(outFolder, opt) {
     opt = {};
   }
 
+  var options = assign({}, opt, {
```

Comment:
this should just become `var sourcemaps = valueOrFunctions(['boolean', 'string', 'object'], opt.sourcemaps);`